study-builder/studies/brain/emails/about_you_completed.html

```diff
@@ -113,27 +113,152 @@
       <p style="font-family: 'Roboto', sans-serif; text-align: left; font-weight: 300; font-size: 15px; line-height: 23px;">Thank
         you for participating. Any insights that we generate will be a direct result of working with you. If you have
         questions at any point and would like to talk to a member of our study staff, please reach out to us at <a
-            href="mailto:info@braincancerproject.org" target="_blank" style="color: #3785BF; text-decoration: none; cursor: pointer;">info@braincancerproject.org</a>
+            href="mailto:info@braincancerproject.org" target="_blank" style="color: #7154ff; text-decoration: none; cursor: pointer;">info@braincancerproject.org</a>
         or call 651-229-3480.</p>
       <p style="font-family: 'Roboto', sans-serif; text-align: left; font-weight: 300; font-size: 15px; line-height: 23px;">Sincerely,</p>
       <p style="font-family: 'Roboto', sans-serif; text-align: left; font-weight: 300; font-size: 15px; line-height: 23px;">The
         Brain Cancer Project Team</p>
       <p style="font-family: 'Roboto', sans-serif; text-align: left; font-weight: 300; font-size: 15px; line-height: 23px;">For
-        additional updates, follow us on Twitter <a href="https://twitter.com/BrainCancerProj" target="_blank" style="color: #3785BF; text-decoration: none; cursor: pointer;">@BrainCancerProj</a>
-        or like our Facebook page, <a href="https://www.facebook.com/braincancerproject" target="_blank" style="color: #3785BF; text-decoration: none; cursor: pointer;">The
+        additional updates, follow us on Twitter <a href="https://twitter.com/BrainCancerProj" target="_blank" style="color: #7154ff; text-decoration: none; cursor: pointer;">@BrainCancerProj</a>
+        or like our Facebook page, <a href="https://www.facebook.com/braincancerproject" target="_blank" style="color: #7154ff; text-decoration: none; cursor: pointer;">The
           Brain Cancer Project</a>.</p>
     </td>
   </tr>
 </table>
+</body>
+
+</html>
+<html>
+<meta name="format-detection" content="telephone=no,date=no,address=no,email=no,url=no" />
+
+<head>
```

Comment:
What's going on here? Why are there two `html` tags in the same file?

Reply:
looks like copy paste error. cleaned up